src/test/java/com/google/cloud/genomics/utils/IntegrationTestHelper.java

```diff
@@ -0,0 +1,108 @@
+    "NA12878",
+    "NA12884",
+    "NA12880",
+  };
+  public static final String[] PLATINUM_GENOMES_READGROUPSETS = {
+    "CMvnhpKTFhCAv6TKo6Dglgg",
+    "CMvnhpKTFhDw8e3V6aCB-Q8",
+    "CMvnhpKTFhDo08GNkfe-jxo",
+    "CMvnhpKTFhD3he72j4KZuyc",
+    "CMvnhpKTFhCIy4KD0qrtjzA",
+    "CMvnhpKTFhCyz5LDrZ7Jozs",
+    "CMvnhpKTFhDyy__v0qfPpkw",
+    "CMvnhpKTFhDIr8bKjdbdolc",
+    "CMvnhpKTFhC1m-r_6Omp5X0",
+    "CMvnhpKTFhCw5I3dqMq6mYMB",
+    "CMvnhpKTFhD_tKnl__-RqJwB",
+    "CMvnhpKTFhDLyY-4kYvurqsB",
+    "CMvnhpKTFhCoyJTFk73Eyq0B",
+    "CMvnhpKTFhDE9a7F7Yai2rAB",
+    "CMvnhpKTFhCUpIDDveWE-r0B",
+    "CMvnhpKTFhCrvIOEw4Ol__sB",
+  };
+  public static final String PLATINUM_GENOMES_BRCA1_REFERENCES = "chr17:41196311:41277499";
+  public static final ReferenceBound[] PLATINUM_GENOMES_VARIANTSET_BOUNDS = {
+    new ReferenceBound().setReferenceName("chr1").setUpperBound(250226910L),
+    new ReferenceBound().setReferenceName("chr10").setUpperBound(136466007L),
+    new ReferenceBound().setReferenceName("chr11").setUpperBound(135762137L),
+    new ReferenceBound().setReferenceName("chr12").setUpperBound(134049696L),
+    new ReferenceBound().setReferenceName("chr13").setUpperBound(115800144L),
+    new ReferenceBound().setReferenceName("chr14").setUpperBound(107857350L),
+    new ReferenceBound().setReferenceName("chr15").setUpperBound(103000009L),
+    new ReferenceBound().setReferenceName("chr16").setUpperBound(90760361L),
+    new ReferenceBound().setReferenceName("chr17").setUpperBound(81983044L),
+    new ReferenceBound().setReferenceName("chr18").setUpperBound(78776233L),
+    new ReferenceBound().setReferenceName("chr19").setUpperBound(59544813L),
+    new ReferenceBound().setReferenceName("chr2").setUpperBound(243800708L),
+    new ReferenceBound().setReferenceName("chr20").setUpperBound(62993757L),
+    new ReferenceBound().setReferenceName("chr21").setUpperBound(48724643L),
+    new ReferenceBound().setReferenceName("chr22").setUpperBound(51891601L),
+    new ReferenceBound().setReferenceName("chr3").setUpperBound(198316350L),
+    new ReferenceBound().setReferenceName("chr4").setUpperBound(191970744L),
+    new ReferenceBound().setReferenceName("chr5").setUpperBound(181054248L),
+    new ReferenceBound().setReferenceName("chr6").setUpperBound(171796962L),
+    new ReferenceBound().setReferenceName("chr7").setUpperBound(159737113L),
+    new ReferenceBound().setReferenceName("chr8").setUpperBound(147299246L),
+    new ReferenceBound().setReferenceName("chr9").setUpperBound(142027288L),
+    new ReferenceBound().setReferenceName("chrM").setUpperBound(1000001L),
+    new ReferenceBound().setReferenceName("chrX").setUpperBound(156231278L),
+    new ReferenceBound().setReferenceName("chrY").setUpperBound(60032946L)
+  };
+
+  // Test configuration constants
+  public final String API_KEY = System.getenv("GOOGLE_API_KEY");
+  public final OfflineAuth auth;
+  
+  public IntegrationTestHelper() throws IOException, GeneralSecurityException {
```

Comment:
I like the idea of a test helper. Here there's a risk that users may forget to call the constructor - all methods are static after all, and it's all too easy to write IntegreationTestHelper.API_KEY.

What I'd like to suggest instead is to have either
(a) API_KEY and auth are protected, only accessible via non-static getters. Thus it's impossible to forget to call the constructor (if we need it)
(b) (I prefer this one) API_KEY and auth are static protected, only accessible via static getters. The getters will grab the information on the first call, stash it for later calls. The upside is that if we don't need auth but need something else from the helper, we can still use it without getting an error. The downside is this isn't thread-safe so we either live with that or spend a bit more time making it so.